src/ott/solvers/was_solver.py

```diff
@@ -11,6 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import inspect
```

Comment:
In general, I think there's a slightly better solution rather than inspecting the signature of the linear solvers,
I'd rather make `linear_ot_solver` a required argument and remove the construction of the solver in `__init__` altogether - this will require some changes, esp. in tests, in `ott/solvers/quadratic/_solve.py`, etc.

Reply:
then this means we can also remove `kwargs` right? I'd also prefer this to `inspect` but didn't want to change the interface in case you had other plans.